models/database.go

```diff
@@ -159,6 +159,13 @@ var databaseSchema = [][]string{
 		)`,
 		`INSERT INTO settings (settings) VALUES ('{}')`,
 	},
+
+	3: {
+		`ALTER TABLE agents 
+           ADD COLUMN tls BOOLEAN NOT NULL DEFAULT FALSE,
+           ADD COLUMN tls_skip_verify BOOLEAN NOT NULL DEFAULT TRUE
+		`,
```

Comment:
The idea was to add another `ALTER TABLE` that would remove default values. That way we will migrate existing data, but also will not have `DEFAULT`s in database (as they can be problematic in corner-cases for reform).